xpk.py

```diff
@@ -1392,28 +1838,51 @@ def run_gke_node_pool_create_command(args, system) -> int:
 
   commands = []
   task_names = []
+
+  xpk_print(
+      f'Creating {args.num_slices} node pool or pools of {system.device_type}\n'
+      f'Underlyingly, we assume that means: {system}'
+  )
+
+  if system.device_type == h100_device_type:
+    desired_node_pool_names = [f'{args.cluster}-np-0']
```

Comment:
Oh this is interesting, do h100s only have one np? If each slice is in a different NP then the normal logic should work right?

Also is this gpu specific or h100 specific

Reply:
This is what we learnt from the GKE doc, both A3 and A3+ have this feature.  (https://docs.google.com/document/d/1ijw1hiAvgNQk6ug8BBsjA2a3opnAM1jHsqdLCAQerTM/edit?pli=1&resourcekey=0-k05v3cpWnx8r0DtZVgfdqw&tab=t.0#heading=h.6wmhdex26cnj):

When creating the A3 node-pool, you could optionally choose to create a nodepool with Kernel Crash Dump enabled, which could capture a full kernel memory crash dump in the event of a kernel crash and save it locally on the instance’s boot disk.

Reply:
Talking offline, we figured that GPUs will use one node pool.